From 720ee83e860431dddefedf5d903380629327ca83 Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@lineageos.org>
Date: Wed, 13 Feb 2019 17:03:23 +0200
Subject: [PATCH 5/5] legacy: Label /data/misc/display again

* Apparently it's still required for legacy mm-pp-daemon in order
  to restore chosen color profile on boot

Change-Id: I0c4daace2e1045a87b9617949e2da9e577c2793c
---
 legacy-common/file_contexts | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/legacy-common/file_contexts b/legacy-common/file_contexts
index ffc8818..a381828 100644
--- a/legacy-common/file_contexts
+++ b/legacy-common/file_contexts
@@ -97,3 +97,6 @@
 # CPU/perf
 /sys/module/msm_pm/modes/cpu[0-9](/.*)?                                       u:object_r:sysfs_msm_perf:s0
 /sys/devices/qcom,cpubw\.[0-9]+/devfreq/qcom,cpubw\.[0-9]+/governor           u:object_r:sysfs_devfreq:s0
+
+# Postprocessing
+/data/misc/display(/.*)?                                                      u:object_r:display_vendor_data_file:s0
-- 
2.17.1

